examples/todos/containers/App.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { addTodo } from '../actions'
+import AddTodo from '../components/AddTodo'
+import Footer from '../components/Footer'
+import VisibleTodoList from './VisibleTodoList'
+import FilterLink from './FilterLink'
+
+const App = ({ dispatch }) => (
```

Comment:
@tracker1,  Thanks for the comment. I personally would like to test the code under containers with Integration Test(or Feature Test). But in this PR, I am trying to focus to demo a potential better way to make an app more unit testable.